cpu/lpc23xx/mci/lpc23xx-mci.c

```diff
@@ -509,20 +510,20 @@ diskio_sta_t mci_initialize(void)
 
     power_off();
 
-    xtimer_usleep(1000);
+    ztimer_sleep(ZTIMER_MSEC, 1);
 
     power_on();                             /* Force socket power on */
     MCI_CLOCK = 0x100 | (PCLK / MCLK_ID / 2 - 1);   /* Set MCICLK = MCLK_ID */
     //for (Timer[0] = 2; Timer[0]; );
-    xtimer_usleep(250);
+    ztimer_sleep(ZTIMER_MSEC, 1);
 
     send_cmd(CMD0, 0, 0, resp);             /* Enter idle state */
     CardRCA = 0;
 
     /*---- Card is 'idle' state ----*/
 
     /* Initialization timeout of 1000 msec */
-    uint32_t start = xtimer_now_usec();
+    uint32_t start = ztimer_now(ZTIMER_MSEC);
```

Comment:
Just to clarify: `ztimer_now()` will always work for `ZTIMER_MSEC`, but not for `ZTIMER_USEC` right?

Reply:
I don't think I get your question...

Reply:
Basically `ztimer_now()` will only return ascending values if the underlying clock is running, see #17298

AFAIU this is only given if a timer is set on the clock. See #17229

Reply:
Ah yes... that is an issue is was not fully aware of either. It will depnd on many things I guess, which makes the issue even more relevant:

- is it backed by `rtt` or `periph_timer`
- is `CONFIG_ZTIMER_USEC_REQUIRED_PM_MODE` set to something (which is not the case)
- is `pm_layered` used

Since this CPU as not rtt, the first item can not be true...

Reply:
@benpicco after further investigating I changed my stance on this, in the default setup `ztimer_now()` is not broken, see https://github.com/RIOT-OS/RIOT/issues/17298#issuecomment-994585430.

Reply:
Then I wonder: If this CPU has no MSEC timer / RTT, why not stay with USEC?

Reply:
Switched to usec